source/src/molecule-unfolding/cdk/utils/custom-resource-lambda/create-event-rule/index.js

```diff
@@ -0,0 +1,114 @@
+const {
+    CloudFormationClient,
+    CreateStackCommand,
+    UpdateStackCommand,
+    DeleteStackCommand,
+    DescribeStacksCommand
+} = require("@aws-sdk/client-cloudformation");
+
+const {
+    readFileSync
+} = require('fs');
+
+exports.handler = async function (event, context) {
+    await _handler(event, context).then(() => {
+        console.log("=== complete ===")
+    }).catch((e) => {
+        console.log(e);
+        throw e
+    })
+}
+
+function sleep(millis) {
+    return new Promise(resolve => setTimeout(resolve, millis));
+}
+
+async function _handler(event, context) {
```

Comment:
I would like to see use TS for strong typing programming. And use [lambda type](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/aws-lambda) to simplify the **CustomResource** event.

Reply:
changed to typescript